app/templates/home.hbs

```diff
@@ -33,14 +33,9 @@
         <div class="sidebar">
           <p>Popular Tags</p>
           <div class="tag-list">
-            <a href="" class="tag-pill tag-default">programming</a>
-            <a href="" class="tag-pill tag-default">javascript</a>
-            <a href="" class="tag-pill tag-default">emberjs</a>
-            <a href="" class="tag-pill tag-default">angularjs</a>
-            <a href="" class="tag-pill tag-default">react</a>
-            <a href="" class="tag-pill tag-default">mean</a>
-            <a href="" class="tag-pill tag-default">node</a>
-            <a href="" class="tag-pill tag-default">rails</a>
+            {{#each model.popularTags as |tag|}}
```

Comment:
It is also possible to use setupController to set these so that they aren't all on Model.
This is what I will be doing from now.